openssl/src/pkey_ctx.rs

```diff
@@ -489,11 +488,17 @@ impl<T> PkeyCtxRef<T> {
     ///
     /// Requires OpenSSL 1.1.1 or newer.
     #[corresponds(EVP_PKEY_CTX_set_hkdf_mode)]
-    #[cfg(ossl111)]
+    #[cfg(any(ossl110, boringssl))]
     #[inline]
     pub fn set_hkdf_mode(&mut self, mode: HkdfMode) -> Result<(), ErrorStack> {
         unsafe {
-            cvt(ffi::EVP_PKEY_CTX_set_hkdf_mode(self.as_ptr(), mode.0))?;
+            cfg_if::cfg_if! {
+                if #[cfg(boringssl)] {
+                    cvt(ffi::EVP_PKEY_CTX_hkdf_mode(self.as_ptr(), mode.0))?;
```

Comment:
Why did boringssl decide to rename this?

I prefer these kind of things be handled as renames on `use`.

Reply:
I think boringssl just hasn't caught up to https://github.com/openssl/openssl/commit/ed576acdf591d4164905ab98e89ca5a3b99d90ab yet. Will change to `use`.

Reply:
Done